rules.module

```diff
@@ -21,10 +22,30 @@ function rules_help($route_name, RouteMatchInterface $route_match) {
   switch ($route_name) {
     case 'entity.rules_reaction_rule.collection':
       $output = t('Reaction rules, listed below, react on selected events on the site. Each reaction rule may fire any number of <em>actions</em>, and may have any number of <em>conditions</em> that must be met for the actions to be executed. You can also set up <a href=":url1">components</a> – stand-alone sets of Rules configuration that can be used in Rules and other parts of your site. See <a href=":url2">the online documentation</a> for an introduction on how to use Rules.', [
-        ':url1' => \Drupal::url('entity.rules_component.collection'),
+        ':url1' => Url::fromRoute('entity.rules_component.collection'),
         ':url2' => 'http://drupal.org/node/298480',
       ]);
       return $output;
+
+    case 'help.page.rules':
+      $output = '<h3>' . t('About') . '</h3>';
+      $output .= '<p>';
+      $output .= t('The Rules module allows site administrators to define conditionally executed actions based on occurring events (ECA-rules). For more information, see the <a href=":url1" target="_blank">online documentation for the Rules module</a> and the current <a href=":url2" target="_blank">Rules documentation for Drupal 8 on Gitbooks</a>.', [
+        ':url1' => 'https://www.drupal.org/project/rules',
+        ':url2' => 'http://thefubhy.gitbooks.io/rules/content/',
+      ]);
+      $output .= '</p>';
+      $output .= '<p>' . t('The newest source code is available on <a href=":url" target="_blank">Github</a>.', [':url' => 'https://github.com/fago/rules']) . '</p>';
```

Comment:
That is not true, the newest source code is available on drupal.org and Github (mirrored). I think we can just remove that sentence.